frontend/src/pages/pipelines/create/CreatePipeline.tsx

```diff
@@ -1041,6 +993,48 @@ const Content: React.FC<ContentProps> = (props: ContentProps) => {
   // Monitor Region Changed and validate Service Available
   useEffect(() => {
     if (pipelineInfo.region) {
+      if (!update) {
```

Comment:
Let's add a new test case for changing the region when creating the pipeline.

Reply:
It depends on the refactoring mechanism of testing frontend. Let include this case in the future refactoring.